unstructured_ingest/v2/processes/connectors/pinecone.py

```diff
@@ -94,6 +94,10 @@ class PineconeUploaderConfig(UploaderConfig):
     pool_threads: Optional[int] = Field(
         default=1, description="Optional limit on number of threads to use for upload"
     )
+    namespace: Optional[str] = Field(
```

Comment:
I would expect this to show up in the --help but it doesn't. Why is that?

Reply:
I can see it when I print out the `--help` option:
```python
PYTHONPATH=. python unstructured_ingest/main.py s3 pinecone --help
...
Options:
  --batch-size FLOAT              Optional number of records per batch. Will
                                  otherwise limit by size.
  --pool-threads FLOAT            Optional limit on number of threads to use
                                  for upload  [default: 1]
  --namespace TEXT                The namespace to write to. If not specified,
                                  the default namespace is used
...
```

Reply:
can you try `unstructured-ingest local pinecone --help` which is what I think was the traditional way.

Reply:
looks like maybe the cli style help only hits V1?

Reply:
You need to install the cli fresh from the PR to see that:
```shell
pip install .
unstructured-ingest local pinecone --help
Usage: unstructured-ingest local pinecone [OPTIONS]

Options:
  --batch-size FLOAT              Optional number of records per batch. Will
                                  otherwise limit by size.
  --pool-threads FLOAT            Optional limit on number of threads to use
                                  for upload  [default: 1]
  --namespace TEXT                The namespace to write to. If not specified,
                                  the default namespace is used
...
```

Reply:
damn. theres always something. I made a fresh venv to test something and forgot that part. Thanks!